### PR TITLE
add costing docs for evaporator, compressor, heat exchanger, heater/c…

### PR DESCRIPTION
**Pull request by ElmiraShamlou**
Addresses https://github.com/watertap-org/watertap/issues/1360 for :

- evaporator
-  compressor
- heat exchanger
- heater/chiller
- membrane distillation

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/costing/compressor.rst [hunk 1]
```diff
@@ -9,51 +9,49 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 ++++++++++++++++++++++++
 
-The following variables are constructed on the unit block (e.g., m.fs.unit.costing) when applying the `cost_compressor` costing method in the ``watertap_costing_package``:
+The following variables are used on the unit block (e.g., m.fs.unit.costing) when applying the `cost_compressor` costing method in the ``watertap_costing_package``:
 
 .. csv-table::
    :header: "Description", "Symbol", "Variable Name", "Index", "Units"
 
-   "description", ":math:`Symbol_{example}`", "``variable_name``", "[t]", ":math:`\text{dimensionless}`"
+   "Vapor flow rate", ":math:`\dot{m}_{\text{Vap, H2O}}`", "``flow_mass_phase_comp``", "None", ":math:`\text{kg/s}`"
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/compressor.rst]:
```suggestion
   "Vapor flow rate", ":math:`\dot{m}_{\text{Vap, H2O}}`", "``flow_mass_phase_comp``", "[t]", ":math:`\text{kg/s}`"
```

File: docs/technical_reference/costing/evaporator.rst [hunk 2]
```diff
@@ -9,51 +9,40 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 ------------------
 
 * :mod:`watertap.costing.unit_models.evaporator`
 
 References
 ----------
-Aim to include at least one reference in most cases, but delete this section if no references used for cost relationships/default values
\ No newline at end of file
+
+Estimated from multiple sources.
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/evaporator.rst]: Do we know what these sources are?

File: docs/technical_reference/costing/heat_exchanger.rst [hunk 3]
```diff
@@ -9,51 +9,39 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 ------------------
 
 * :mod:`watertap.costing.unit_models.heat_exchanger`
 
 References
 ----------
-Aim to include at least one reference in most cases, but delete this section if no references used for cost relationships/default values
\ No newline at end of file
+
+Estimated from multiple sources.
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/heat_exchanger.rst]: Do we know what these sources are?

File: docs/technical_reference/costing/heater_chiller.rst [hunk 4]
```diff
@@ -9,51 +9,60 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 ++++++++++++++++++++++++
 
-The following variables are constructed on the unit block (e.g., m.fs.unit.costing) when applying the `cost_heater_chiller` costing method in the ``watertap_costing_package``:
+The following variables are used on the unit block (e.g., m.fs.unit.costing) when applying the `cost_heater_chiller` costing method in the ``watertap_costing_package``:
 
 .. csv-table::
    :header: "Description", "Symbol", "Variable Name", "Index", "Units"
 
-   "description", ":math:`Symbol_{example}`", "``variable_name``", "[t]", ":math:`\text{dimensionless}`"
+   "Heat duty", ":math:`E`", "``heat_duty``", "None", ":math:`\text{W}`"
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/heater_chiller.rst]:
```suggestion
   "Heat duty", ":math:`E`", "``heat_duty``", "[t]", ":math:`\text{W}`"
```

File: docs/technical_reference/costing/heater_chiller.rst [hunk 5]
```diff
@@ -9,51 +9,60 @@ The following parameters are constructed for the unit on the FlowsheetCostingBlo
...
 ------------------
 
 * :mod:`watertap.costing.unit_models.heater_chiller`
 
 References
 ----------
-Aim to include at least one reference in most cases, but delete this section if no references used for cost relationships/default values
\ No newline at end of file
+
+- Estimated from multiple sources.
```
Comment (MarcusHolly) [inline docs/technical_reference/costing/heater_chiller.rst]: Do we know what these are? If not, we can probably just delete these sections

**Review by MarcusHolly — COMMENTED** [review]
LGTM - just a couple of minor comments about indices and references.

File: docs/technical_reference/costing/heat_exchanger.rst (hunk 3 above)
Comment (ElmiraShamlou) [reply docs/technical_reference/costing/heat_exchanger.rst]: deleted.

File: docs/technical_reference/costing/evaporator.rst (hunk 2 above)
Comment (ElmiraShamlou) [reply docs/technical_reference/costing/evaporator.rst]: deleted.

Comment (MarcusHolly) [reply docs/technical_reference/costing/evaporator.rst]: I think you still need to push the change deleting these

**Review by MarcusHolly — CHANGES_REQUESTED** [review]
Some of the files still have the "Estimated from multiple sources" under the reference section

File: docs/technical_reference/costing/evaporator.rst (hunk 2 above)
Comment (ElmiraShamlou) [inline docs/technical_reference/costing/evaporator.rst]:
```suggestion

```

**Review by MarcusHolly — APPROVED** [review]
LGTM

**Review by adam-a-a — APPROVED** [review]
Great job!

**Pull request merged**